datadog_checks_base/datadog_checks/base/utils/__init__.py

```diff
@@ -8,7 +8,12 @@
     if datadog_agent.get_config('integration_tracing'):
         from ddtrace import patch
 
-        patch(requests=True)
+        # handle thread monitoring as an additional option
+        if datadog_agent.get_config('integration_tracing_futures'):
+            patch(requests=True, futures=True)
```

Comment:
Can we add a link to the `dd-trace-py` docs here? At first I was confused because the comment was mentioning threads but the option is named `futures` — that's because the name is misleading in `dd-trace-py` itself (the `futures` extensions deals with propagating trace info across threads, not with the `concurrent.futures` module).

```suggestion
            # See: http://pypi.datadoghq.com/trace/docs/other_integrations.html#futures
            patch(requests=True, futures=True)
```